src/main/java/frc/robot/commands/ArmCommand.java

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants.ArmConstants;
+import frc.robot.subsystems.ArmSubsystem;
+
+public class ArmCommand extends CommandBase {
+
+  private final PIDController m_pivotPIDController = new PIDController(
+      ArmConstants.kPPivotController, 0, 0);
+  private final PIDController m_elevatorPIDController = new PIDController(
+      ArmConstants.kPElevatorController, 0, 0);
+
+  private ArmSubsystem m_armSubsystem;
+
+  private double m_pivotSetpoint;
+  private double m_elevatorSetpoint;
+
+  /** Creates a new DriveElevatorCommand. */
+  public ArmCommand(ArmSubsystem armSubsystem, double pivotSetpoint, double elevatorSetpoint) {
+    // Use addRequirements() here to declare subsystem dependencies.
+    m_armSubsystem = armSubsystem;
+    addRequirements(armSubsystem);
+
+    m_pivotSetpoint = pivotSetpoint;
```

Comment:
intead of having `m_pivotSetpoint` and `m_elevatorSetpoint` just call `PIDController.setSetpoint`